fargate/metadata.csv

```diff
@@ -0,0 +1,24 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+fargate.io.ops.write,rate,,,,# write operations to the disk,0,fargate,io write count
+fargate.io.bytes.write,rate,,byte,,# bytes written to the disk,0,fargate,io write
+fargate.io.ops.read,rate,,,,# read operation on the disk,0,fargate,io read count
+fargate.io.bytes.read,rate,,byte,,# bytes read on the disk,0,fargate,io read
+fargate.cpu.user,rate,,nanosecond,,cpu time spent in ns,-1,fargate,cpu user
```

Comment:
Orientation should be 0 imo. For a container that is supposed to consume a lot of CPU time, CPU usage going down is likely a problem, not a good thing.